src/main/kotlin/Checker.kt

```diff
@@ -0,0 +1,25 @@
+class Checker(val secretCode: List<String>) {
+
+    fun checkGuess(guess: List<String>): Feedback{
```

Comment:
```suggestion
    fun checkGuess(guess: List<String>): Feedback {
```
nie było formatowania